realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -881,6 +881,108 @@ public void contains_caseSensitiveWithNonLatinCharacters() {
     }
 
     @Test
+    public void like_caseSensitive() {
+        final int TEST_OBJECTS_COUNT = 200;
+        populateTestRealm(realm, TEST_OBJECTS_COUNT);
+
+        RealmResults<AllTypes> resultList = realm.where(AllTypes.class)
+                .like("columnString", "*DaTa*").findAll();
```

Comment:
Minor: If everything doesn't fit on one line, it would probably put `.findAll()` on a new line for better readability (and since queries usually read like a builder).